internal_transfers/actions/src/simulate/interface.ts

```diff
@@ -28,4 +30,10 @@ export abstract class TxSimulator {
     save: boolean,
     blockNumber?: number
   ): Promise<any>;
+
+  /**
+   * Converts simulation result into `SimulationData` containing only the relevant fields.
+   * @param simulation - return value from simulate method.
+   */
+  abstract parseSimulation(simulation: any): SimulationData;
```

Comment:
it would be great if the param is not of type any

Reply:
I don't think this method belong to the abstract class. 

It should be a static util, used by Tenderly implementation of the interface I suggested in https://github.com/cowprotocol/solver-rewards/pull/227

Instead, the method `simulate` should return Promise<SimulationData> if this is the only data you care about. So you can start with simpler closed interfaces, and add data to them based on need.

If in the future you need more data in `SimulationData` you just add it, and then the static function i suggested above will require to implement the extraction of the data from the Tenderly result

Reply:
BTW, parse is usually applied if you are dividing some string, bytes, into some and extracting some logical parts. 

Here you are technically converging one object into another one. So i don't think you are parsing anything

Reply:
"Parse" is to resolve something into its parts. I would say that this is exactly what we are doing here. Pruning irrelevant pieces and interpreting the response.